changelogs/fragments/65993-restart-docker-container-on-restart-policy-updates.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
Please rename the file so it contains `docker_container` in its name (and not `docker-container`).

Reply:
Done.